diracx-core/src/diracx/core/config/schema.py

```diff
@@ -159,6 +159,56 @@ class OperationsConfig(BaseModel):
     ResourceStatus: dict[str, Any] | None = None
 
 
+class DBConfig(BaseModel):
+    DBName: str
+    Host: str
+    Port: int
+
+
+class JobDBConfig(DBConfig):
+    MaxRescheduling: int = 3
+
+
+class DatabasesConfig(BaseModel):
+    JobDB: JobDBConfig
+    JobLoggingDB: DBConfig
+    PilotAgentsDB: DBConfig
+    SandboxMetadataDB: DBConfig
+    TaskQueueDB: DBConfig
+    ElasticJobParametersDB: DBConfig
+    VirtualMachineDB: DBConfig
+
+
+class ProductionConfig(BaseModel):
+    URLs: dict[str, Any] | None = None
+    Services: dict[str, Any] | None = None
+    Agents: dict[str, Any] | None = None
+    JobWrapper: dict[str, Any] | None = None
+    Databases: DatabasesConfig
+    Executors: dict[str, Any] | None = None
+    FailoverURLs: dict[str, Any] | None = None
+
+
+class WorkloadManagementConfig(BaseModel):
+    Production: ProductionConfig | None = None
+
+
+class SystemsConfig(BaseModel):
+    Accounting: dict[str, Any] | None = None
+    Bookkeeping: dict[str, Any] | None = None
+    Configuration: dict[str, Any] | None = None
+    Framework: dict[str, Any] | None = None
+    DataMangement: dict[str, Any] | None = None
+    Monitoring: dict[str, Any] | None = None
+    ProductionManagement: dict[str, Any] | None = None
+    RequestManagement: dict[str, Any] | None = None
+    ResourceStatus: dict[str, Any] | None = None
+    StorageManagement: dict[str, Any] | None = None
+    Transformation: dict[str, Any] | None = None
+    WorkloadManagement: WorkloadManagementConfig | None = None
```

Comment:
And therefore change this line:

```suggestion
    WorkloadManagement: ProductionConfig | None = None
```

May be `ProductionConfig` is not the most appropriate name here now though. What about having a generic `SystemConfig` with `URLS, Services, Agents, Databases, Executors, FailoverURLs` and a `WorkloadManagementConfig` inheriting from `SystemConfig` with `JobWrapper`?